staging/cert-manager-setup/templates/clusterissuer.yaml

```diff
@@ -0,0 +1,47 @@
+# DEPRECATED, this file should be deleted soon
```

Comment:
Can we please create/use a JIRA here and link the follow up?

Reply:
will do, but not sure what we need to do atm.